articles/user-profile/normalized/auth0/index.md

```diff
@@ -2,6 +2,10 @@
 description: Auth0 normalizes common user properties in the User Profile.
 ---
 
+::: version-warning
+The user profile returned after successful authentication using an OIDC-compliant flow may differ from the Auth0 normalized profile. Please use the toggle to see the claims available on that profile.
+:::
+
```

Comment:
Also, you need to update this paragraph https://github.com/auth0/docs/pull/4623/files#diff-0aeba265d924efd23d617f971e37b079R196 : 
- The Authentication API /userinfo endpoint, returns the OIDC user profile, not the normalized
- The User Profile section of our quickstarts, uses methods that return the OIDC user profile, not the normalized
- The way to retrieve the normalized user profile is using the Management API https://auth0.com/docs/api/management/v2#!/Users/get_users_by_id
- Don't know if you can retrieve the normalized user profile using Lock or auth0.js (Call the management API endpoints using these libraries? Is there another way? Cross check)